lib/commands/run.ts

```diff
@@ -32,9 +32,9 @@ export class RunCommandBase implements ICommand {
 			this.platform = this.$devicePlatformsConstants.Android;
 		}
 
-		const availablePlatforms = this.platform ? [this.platform] : this.$platformsData.availablePlatforms;
-		for (let platform of availablePlatforms) {
-			const platformData = this.$platformsData.getPlatformData(platform, this.$projectData);
+		const availablePlatforms = this.platform ? {[this.platform]: this.platform} : this.$platformsData.availablePlatforms;
```

Comment:
I'd like to suggest a different approach to the problem - see if it suits you:
Instead of changing the array `[this.platform]` into an object why not change the object into an array.
That way the code would look like this:
```TypeScript
const availablePlatforms = this.platform ? [this.platform] : _.values(this.$platformsData.availablePlatforms);

for (const platform of availablePlatforms) {
```

That way you can retain the `for ... of` loop and it would be a bit more readable than using an object.
@KristianDD what do you think?